### ISSUE TITLE
Bash don't work inside VS Code, cannot type 'Shell integration failed to activate'

### ISSUE THREAD

**Issue by 3mper0r** [microsoft/vscode#200231]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: latest
- OS Version: Linux Mint 20

1. My Bash shell is not working inside VS Code, I simply cannot type anything when I'm inside VS Code but I can when I open the terminal outside VS Code. This happened after I installed Node's version 20 and took the latest version of VS Code. In the info bar in the terminal icon when hovered it shows "Shell integration failed to activate"

**Comment by 3mper0r** [comment]
Thanks for sharing the link, I went through all the docs, also the other 5 subcategories in Terminal section, no help from there.

What I tried:
- uninstalled the whole VS CODE (cache included) with all extensions, installed again, same thing, same happened with vscode-insiders (even though installed for the first time)
- updated the bash shell version, bash shell comes as a default shell in linux mint
- installed powershell (pwsh) and zsh and are working fine inside vsCode
- terminal with bash being as a default works fine outside vsCode
- bash works fine inside Atom (code editor) under the same path

I somehow think there is a conflict in vscode terminal configs

**Comment by Tyriar** [comment]
To see if it's shell integration that's causing the problem, could you try setting `"terminal.integrated.shellIntegration.enabled": false`? There may be some conflict in your bash setup that breaks the script and you might need to install manually https://code.visualstudio.com/docs/terminal/shell-integration#_manual-installation

**Comment by 3mper0r** [comment]
I have tried this 4days ago already, basically I went on toggling all the configs in @feature:terminal inside vsCode one by one, none did help, bash opens but all black(no root path shown) and you cannot type anything. 

I spent a whole week debugging on this and just now I accidentally solved this mysterious issue when I just wanted to remove my git, the one which I have initialized in my root folder accidentally some years ago, so I click show hide files to remove the .git folder and then somewhere close my eye catches files related to our cause such .bashrc , .bash_profile, .bash_logut and .bash_history so I remove all of them and magic bash comes back to life + without restarting the vsCode or terminal.

Moral of the story: when you try long enough, things happen even if you don't get it how it actually happened!
Obviously path/bash scripts inside profile/bashrc got mixed up somewhere in the road even though I didn't touch it, the only thing I did was upgrading Node to the latest before this occurred (also nvm commands stopped working)!

Anyway thanks a lot Daniel!!!
